platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/cli/CleanCommand.java

```diff
@@ -66,7 +66,7 @@ public static void clean(@NonNull final Path sdkPath) throws IOException {
         Objects.requireNonNull(sdkPath);
 
         final Configuration configuration = DefaultConfiguration.buildBasicConfiguration(
-                List.of(sdkPath.resolve("config.txt"), sdkPath.resolve("settings.txt")));
+                sdkPath.resolve("settings.txt"), List.of(sdkPath.resolve("config.txt")));
```

Comment:
Can we put these file names in a constant somewhere and use that constant? I see these strings all over the place

Reply:
Created constants.